target/ppc/mmu-hash64.c

```diff
@@ -180,7 +180,7 @@ void helper_SLBIAG(CPUPPCState *env, target_ulong rs, uint32_t l)
     int n; 
 
     /*
-     * slbia must always flush all TLB (which is equivalent to ERAT in ppc
+     * slbiag must always flush all TLB (which is equivalent to ERAT in ppc
      * architecture). Matching on SLB_ESID_V is not good enough, because slbmte
      * can overwrite a valid SLB without flushing its lookaside information.
      *
```

Comment:
There are still some mentions to slbia, below, in this comment block.

Reply:
Fixed in [eadbe0d](https://github.com/PPC64/qemu/pull/89/commits/eadbe0da6549ca876b044e312a3e0ca915e34ef3)